.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: go
 
 go:
-  - "1.11.x"
-  - "1.12.x"
   - "1.13.x"
```

Comment:
it's better to add 1.16 and 1.18 checking.

Reply:
perfect